src/components/Moon.jsx

```diff
@@ -6,34 +6,25 @@ class Moon extends React.Component {
     constructor(props) {
         super(props);
 
+        this.getData = this.getData.bind(this);
+        this.getDate = this.getDate.bind(this);
+
         this.state = {
             error: null,
             data: null,
             isLoading: false,
-            timer: new Date(),
         };
-    }
+    };
 
     componentDidMount() {
-        this.timerID = setInterval(
-            () => this.tick(),
-            1000
-        );
-
-        this.setState({ isLoading: true })
+        this.setState({ isLoading: true });
         this.getData();
-
-    }
+        setInterval(this.getData, 600000);
```

Comment:
600,000 milliseconds is 600 seconds (10 minutes). I don't remember if you wanted this to fire every minute or not. If so, it might be better to use 60*1000.

Reply:
This is correct, I believe 10 minutes should be a fair amount of time to get the next set of data.